2.0/ingest/ingest_logic.py

```
@@ -0,0 +1,67 @@
+            stmt = stmt.on_conflict_do_nothing(index_elements=['accession_number']) 
+            try:
+                conn.execute(stmt)
+            except Exception as e:
+                logging.error(f"Failed to insert row {row['accession_number']}: {e}")
+
+def ingest_dataframe(df, table_name, engine):
+    df = serialize_nonbasic_df_columns(df)
+
+    try:
+        df.to_sql(name=table_name, con=engine, if_exists='append', index=False, method='multi')
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)